src/ansible_navigator/actions/sample_form.py

```diff
@@ -99,9 +99,8 @@ def run(self, interaction: Interaction, app: AppPublic) -> Interaction:
         """Handle :doc
 
         :param interaction: The interaction from the user
-        :type interaction: Interaction
         :param app: The app instance
-        :type app: App
+        :return: The pending :class:`interaction <~Interaction>`
```

Comment:
`~` only makes sense when you don't override the anchor. If you were to do
```rst
:class:`~some.subpkg.Interaction`
```
that would be an equivalent of
```rst
:class:`Interaction <some.subpkg.Interaction>`
```
but with the way you use it, I'm not even sure it'd work.
The proper thing would be something like
```suggestion
        :return: The pending :class:`interaction <ansible_navigator.actions.sample_form.Interaction>`
```
(or whatever the full correct identifier is, I haven't checked)

Reply:
good suggestion, the change was made across all the actions

Reply:
here too, a link to the docs for an `Interaction` 
![image](https://user-images.githubusercontent.com/18386516/150585483-ecb72eae-8ce4-4e4c-a71c-cb11c4105899.png)

Reply:
Resolving.